Applications/DataExplorer/mainwindow.cpp

```diff
@@ -430,10 +430,10 @@ void MainWindow::save()
         std::vector<std::string> selected_geometries;
         _project.getGEOObjects().getGeometryNames(selected_geometries);
 
-        GMSHInterface gmsh_io(
+        GeoLib::IO::GMSHInterface gmsh_io(
             _project.getGEOObjects(), true,
-            FileIO::GMSH::MeshDensityAlgorithm::AdaptiveMeshDensity,
-            0.05, 0.5, 2, selected_geometries);
+            GeoLib::IO::GMSH::MeshDensityAlgorithm::AdaptiveMeshDensity, 0.05,
```

Comment:
The directory is named GeoLib/IO/GmshIO, but the namespace is GeoLib::IO::GMSH. Maybe the two things should match?